packages/cascara/src/modules/ActionEdit/ActionEdit.doc.mdx

```diff
@@ -1,11 +1,101 @@
 ---
-title: ActionEdit
+title: Action Edit
 propTable: ActionEdit.js
 type: 'module'
 status: 'Draft'
 public: false
 ---
 
+A special action module, fully controled by **esp-auth**. Use it when you want the data to be editable.
+In contrast with the **button** module, this module renders up to 2 buttons, depending on the **edit** state of the parent context:
+
+1. when not in **edit** mode, it will render a single button with the text you pass as **editLabel** prop, clicking that button will make the **edit** mode kick-in
+2. when in **edit** mode, it will render a button for exiting **edit** mode (AKA cancel) and one for accepting the changes made during **edit** mode (AKA saving).
+
+All these three buttons will cause your Table or Form to emit the following **onAction** events:
+
+1. **edit.start** when entering in **edit** mode
+2. **edit.cancel** when exiting the **edit** mode via the **cancel** button
+3. **edit.save** when exiting the **edit** mode via the **save** button
+
+## Using the module
+
+Additional to the props specified in the aside table, you need to pass the `module` prop, with its value set to **edit**.
+
+### Playground
+
+Feel free to play around with the following code:
+
+```jsx
+<ModuleExample>
+  <ActionEdit cancelLabel='Cancelar' editLabel='Editar' saveLabel='Guardar' />
+</ModuleExample>
 ```
-<ActionEdit />
+
+### Use in Table
+
+Adding this module to your **Cascara Table** would look like this:
+
+```jsx
+<Table
+  // define the actions we want to appear for each table row
+  actions={{
+    actionButtonMenuIndex: -1,
+    modules: [
+      {
+        module: 'edit',
+
+        // any other valid props
+      },
+    ],
+  }}
+  // your data
+  data={[
+    {
+      id: 1,
+      userEmail: 'john@doe.it',
+      userName: 'John Doe',
+    },
+  ]}
+  // the fields in our data you want to display
+  dataConfig={{
+    display: [
+      {
+        attribute: 'id',
+        isEditable: false,
+        isLabeled: false,
+        label: 'ID',
+        module: 'number',
+      },
+      {
+        attribute: 'userEmail',
+        isEditable: true,
+        isLabeled: false,
+        label: 'eMail',
+        module: 'text',
+      },
+      {
+        attribute: 'userName',
+        isEditable: true,
+        isLabeled: false,
+        label: 'Name',
+        module: 'text',
+      },
+    ],
+  }}
+  // the action handler is invoken when a button is clicked
+  onAction={(action, record) => {
+    console.clear();
+    console.log(`Action ${action.name} invoked`);
+    console.table(record);
+
+    switch (action.name) {
+      case 'edit.start':
+        console.log(`Your action '${action.name}' has been invoked!`);
+        break;
+    }
+  }}
+  // this defines the unique ID field in your data
+  uniqueIdAttribute={'id'}
+/>
```

Comment:
Before we are done, I think all of these examples with Table should come out of the MDX files. This is still a lot of boilerplate to maintain. I assume we are just leaving them here for now while we validate the sandbox example you put together for modules.

Reply:
I wanted to keep Table examples, since that is the way developers are going to use it. The `ModulePlayground` shows  the modules in isolation, but the Table examples show how modules are ultimately used.

- [ ] Remove Table examples from module docs